modules/deployments-rolling-strategy.adoc

```diff
@@ -60,3 +60,9 @@ These parameters allow the deployment to be tuned for availability and speed. Fo
 
 Generally, if you want fast rollouts, use `maxSurge`. If you have to take into account resource quota and can accept partial unavailability, use
 `maxUnavailable`.
+
+[WARNING]
+====
+The default setting for `maxUnavailble` is `1` for all the machine config pools in {product-title}. Do not change this value to `3` for the control plane pool.
```

Comment:
Just verifying that it's `maxUnavailble` and not `maxUnavailable`? (applies to all of these files)

Reply:
LOL no you're right, i just misspelled it. good catch!

Reply:
ok i think i got them all. thanks @kcarmichael08 !